src/Engine/Graphics/Indoor.cpp

```diff
@@ -1305,6 +1307,21 @@ void UpdateActors_BLV() {
         if (actor.vPosition.z > floorZ + 1)
             isAboveGround = true;
 
+        // make bloodsplat when the ground is hit
+        if (!actor.donebloodsplat) {
+            if (actor.uAIState == Dead || actor.uAIState == Dying) {
+                if (actor.vPosition.z < floorZ + 30) {
+                    if (pMonsterStats->pInfos[actor.pMonsterInfo.uID].bQuestMonster & 1) {
```

Comment:
Btw, is bQuestMonster even used for anything other than bloodsplats?

Reply:
Doesn't appear to be - will rename.